src/diagnostic.c

```diff
@@ -260,6 +260,7 @@ static const pm_diagnostic_data_t diagnostic_messages[PM_DIAGNOSTIC_ID_LEN] = {
     [PM_ERR_RESCUE_TERM]                        = { "expected a closing delimiter for the `rescue` clause", PM_ERROR_LEVEL_FATAL },
     [PM_ERR_RESCUE_VARIABLE]                    = { "expected an exception variable after `=>` in a rescue statement", PM_ERROR_LEVEL_FATAL },
     [PM_ERR_RETURN_INVALID]                     = { "invalid `return` in a class or module body", PM_ERROR_LEVEL_FATAL },
+    [PM_ERR_SINGLETON_FOR_LITERALS]             = { "can't define singleton method for literals", PM_ERROR_LEVEL_FATAL },
```

Comment:
```suggestion
    [PM_ERR_SINGLETON_FOR_LITERALS]             = { "cannot define singleton method for literals", PM_ERROR_LEVEL_FATAL },
```

I know you're just matching the error message from CRuby, but let's try to go with `cannot` to match our own style. If someone complains we can change it back.